### PR TITLE
fix: Adds link state styling iCore open theme

### PR DESCRIPTION
**Pull request by HeleenSG**
- Adds missing states to icore open theme
- Adds additional styling options to link state styling in Manon

### CODE REVIEW

**Comment by ppvg** [comment]
Would you be opposed to changing the focus outline to 2 pixels? There's no legal requirement yet to comply with WCAG 2.2 (specifically: [success criterion 2.4.13](https://www.w3.org/TR/WCAG22/#focus-appearance)), but it wouldn't hurt either.

File: manon/footer-navigation-link-focus.scss [hunk 1]
```diff
@@ -10,9 +10,14 @@ body > footer,
...
     a {
       &:focus,
-      /* Testing purposes */
       &.focus {
         @include link.link-and-icon-styling("footer-navigation-link-focus-");
+
+        &:visited {
+          @include link.link-and-icon-styling(
+            "footer-navigation-link-visited-focus-"
+          );
```
Comment (ricklambrechts) [inline manon/footer-navigation-link-focus.scss:19]: Just to be sure, is correct?

- `:focus:visited`
- `:visited:hover`

Comment (HeleenSG) [reply manon/footer-navigation-link-focus.scss:19]: I have used these within the iCore open theme to add the styling as needed. As such I have tested them. 

I've added the variables as well: 
https://github.com/minvws/nl-rdo-manon/blob/5c99b9974ae55056ce052679db681bad8ac5aef4/manon/footer-navigation-link-focus-variables.scss#L14-L18

**Comment by HeleenSG** [comment]
> Would you be opposed to changing the focus outline to 2 pixels? There's no legal requirement yet to comply with WCAG 2.2 (specifically: [success criterion 2.4.13](https://www.w3.org/TR/WCAG22/#focus-appearance)), but it wouldn't hurt either.

I will pick up additional styling in a separate pr and I will discuss this with the designers. But I will take this into account.

**Review by ppvg — APPROVED** [review]

**Pull request merged**